src/app/layout/layout.model.ts

```diff
@@ -0,0 +1,13 @@
+
+export interface AppView {
+    name: string,
+    i18n: string
+}
+
+export enum AppBarViews {
```

Comment:
Is not clear for me for which will be the purpose for this. If it actually will be reusable somewhere besides navbar links?